src/handlers/textHandler.js

```diff
@@ -11,7 +11,8 @@ const commands = {
   '!youtube': cmd.youtubeCommand,
   '!twitter': cmd.twitterCommand,
   '!github': cmd.githubCommand,
-  '!report': cmd.reportCommand
+  '!report': cmd.reportCommand,
+  '!hastebinize': cmd.hastebinizeCommand

```

Comment:
You may want to add the trailing comma so that this line doesn't end up in the git diff next time you add a line.

Reply:
We're using standard.js in this project. Trailing comma is not allowed.